src/App.tsx

```diff
@@ -1,10 +1,16 @@
-import React, { useState, useEffect } from 'react';
-import Test from './Test';
+import React from 'react';
+import StyledTextInput from './UI/StyledTextInput';
 
 const App: React.FC = () => (
     <div className="App">
-        EAZY LEARN
-        <Test />
+        <StyledTextInput
+            placeholder="Логин"
+            type="text"
```

Comment:
Why type text if we use text input???

Reply:
this type is default, why not to use it?